apps/admin-gui/src/app/shared/components/dialogs/create-vo-dialog/create-vo-dialog.component.ts

```diff
@@ -45,10 +47,11 @@ export class CreateVoDialogComponent implements OnInit {
 
   onSubmit() {
     this.loading = true;
-    this.voService.createVoWithName(this.fullNameCtrl.value, this.shortNameCtrl.value).subscribe(() => {
+    this.voService.createVoWithName(this.fullNameCtrl.value, this.shortNameCtrl.value).subscribe(data => {
```

Comment:
I have just a small note to the naming conventions.
You named "data" the object that the method returned. For better code readability I think it would be nicer if we rename it in more "describing way". Because what the method is returning is vo, so why not to name it vo?
```suggestion
    this.voService.createVoWithName(this.fullNameCtrl.value, this.shortNameCtrl.value).subscribe(vo => {
```

Reply:
It's true. Renamed!